src/mrinufft/operators/interfaces/gpunufft.py

```diff
@@ -202,6 +202,17 @@ def _reshape_image(self, image, direction="op"):
                 return image.squeeze().astype(xp.complex64, copy=False).T[None]
             return xp.asarray([c.T for c in image], dtype=xp.complex64).squeeze()
 
+    def set_smaps(self, smaps):
```

Comment:
`set_pinned_smaps` ? But if you prefer to remain consistent with other backends, so be it.

Reply:
I think its better to be consistent